3회차/서주현/0_최빈수구하기.py

```diff
@@ -1,3 +1,23 @@
 import sys
 
 sys.stdin = open("_최빈수구하기.txt")
+
+T = int(input())
+
+for i in range(T):
+    sequence = int(input())
+
+    num = list(map(int, input().split()))
+    numdic = {}
+    for i in num :
+        numdic[i] = 0
+    for i in num :
+        numdic[i] += 1
```

Comment:
`if` 구문을 활용해 `for`문을 한 번만 사용할 수도 있습니다!
```suggestion
    for i in num :
        if i not in numdic:   # i 가 numdic에 없다면 
            numdic[i] = 0     # 초기 값으로 0을 설정해주자!
        numdic[i] += 1
```